iso15118/secc/comm_session_handler.py

```diff
@@ -232,6 +232,17 @@ async def __init__.
 
         await wait_for_tasks(self.list_of_tasks)
 
+    def update_config(self, new_config: dict):
+        for key, value in new_config.items():
+            logger.info(
+                f"SECC Config is updated key = {key}: old value = "
+                f"{self.config.as_dict()[key]} - new value = {value}"
+            )
+        self.config.update(new_config)
```

Comment:
Would it be better to call update() before logging the "updated" message?

Reply:
Done